tests/VolleyManagement.UnitTests/WebApi/Controllers/ScheduleTests.cs

```diff
@@ -245,6 +258,28 @@ public void GetSchedule_TournamentHasFreeDayGame_FreeDayGameIsLast()
             ScheduleViewModelComparer.AssertAreEqual(expected, actual);
         }
 
+        [TestMethod]
+        public void GetSchedule_PlayoffScheme_RoundNamesAreCreated()
+        {
+            // Arrange
+            const int TEST_ROUND_COUNT = 5;
```

Comment:
byte

Reply:
Explain why?

Reply:
for memory less usage 4 bytes `int`  vs 1 byte `byte`

Reply:
this memory difference won't provide any observable behavior before we have 1 000 000 of such tests. but indeed byte is better here. Can you tell me why?